packages/ui5-library-writer/src/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface UI5LibConfig {
+    libraryName: string;
+    namespace: string;
+    framework: 'SAPUI5' | 'OpenUI5';
+    frameworkVersion: string;
+    author: string;
```

Comment:
Why is author mandatory?

Reply:
I've made author and typescript optional.